### PR TITLE
Update set-up-guests.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5090
Added link.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-up-guests.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ By default, guest access is turned off. As the Microsoft 365 or Office 365 admin
 After you turn on guest access, it may take a few hours for the changes to take effect. If a user sees the message "Contact your administrator" when they try to add a guest to their team, it's likely that either guest access hasn't been turned on or the settings aren't effective yet.
 
 > [!IMPORTANT]
-> Turning on guest access depends on settings in Azure Active Directory, Microsoft 365 or Office 365, SharePoint Online, and Teams. For more information, see [Authorize guest access in Teams](Teams-dependencies.md).
+> Turning on guest access depends on settings in Azure Active Directory, Microsoft 365 or Office 365, SharePoint Online, and Teams. For more information, see [Authorize guest access in Teams](Teams-dependencies.md) and [Manage guest access in Microsoft 365 groups](https://docs.microsoft.com/microsoft-365/admin/create-groups/manage-guest-access-in-groups?view=o365-worldwide#manage-groups-guest-access)
```
Comment (JohanFreelancer9) [inline Teams/set-up-guests.md:33]:
```suggestion
> Turning on guest access depends on settings in Azure Active Directory, Microsoft 365 or Office 365, SharePoint Online, and Teams. For more information, see [Authorize guest access in Teams](Teams-dependencies.md) and [Manage guest access in Microsoft 365 groups](https://docs.microsoft.com/microsoft-365/admin/create-groups/manage-guest-access-in-groups?view=o365-worldwide#manage-groups-guest-access).
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
@lindspea - In the Note, there's a link to https://docs.microsoft.com/en-us/microsoftteams/teams-dependencies, which discusses in detail the authorization levels for managing guest access features (including Microsoft 365 Group). Can you add the link to the "Control guest access in Microsoft 365 Groups" section instead? That way, the info is in one place.  Thanks.

**Comment by lindspea** [comment]
@LanaChin Thank you. I'll do that.

**Comment by LanaChin** [comment]
Instead of adding the Manage guest access in Microsoft 365 groups](https://docs.microsoft.com/microsoft-365/admin/create-groups/manage-guest-access-in-groups?view=o365-worldwide#manage-groups-guest-access) link to set-up-guests, add it to the Microsoft 365 Groups section of teams-dependencies. That way, the information about Microsoft 365 Groups  and guest access is in one place.

**Comment by LanaChin** [comment]
See #5323

**Pull request closed without merging**